### DISCUSSION TITLE
Updating new Docker version?

### DISCUSSION THREAD

**Discussion by lnlyssg** [pixelfed/pixelfed#5036] [Support]
I'm currently running [v0.11.13](https://github.com/pixelfed/pixelfed/releases/tag/v0.11.13) in Docker using the new setup as documented in the guide from @jippi at https://jippi.github.io/pixelfed-docs-next/pr-preview/pr-1/running-pixelfed/docker/migrate/v2.0.html. If I want to update to the latest commits (or 11.14 when it's ready) what is the process that needs to be followed? Previously I would have stashed the changes (usually just the .env and docker-compose files), pulled the latest version and then carried out some diff checks etc. as not much would have been changed. With the new set-up I can see there are a large amount of changed and untracked working tree files so my old approach isn't going to work....

**Comment by jippi** [comment]
We plan to fix the pixelfed/pixelfed version this week - until then, I updated my `pr-1` branch on my fork

https://github.com/jippi/pixelfed/pkgs/container/pixelfed

They target latest `staging` plus these fixes https://github.com/pixelfed/pixelfed/pull/5035
